notice_and_comment/settings/base.py

```diff
@@ -21,4 +24,6 @@
 
 # Commenting
 BROKER_URL = 'redis://localhost:6379/0'
-REGS_API_URL = 'https://api.data.gov/regulations/v3/'
+REGS_API_URL = env.get_credential('REGS_API_URL', os.environ.get('REGS_API_URL'))
+REGS_API_KEY = env.get_credential('REGS_API_KEY', os.environ.get('REGS_API_KEY'))
+print("URL: {}, Key: {}".format(REGS_API_URL, REGS_API_KEY))
```

Comment:
Stray print here. Would you mind adding dummy values (or using `os.environ.get`) for `ACCESS_KEY_ID`, `SECRET_ACCESS_KEY` and `BUCKET`? It helps w/ documentation. If you go the `os.environ.get` route, I think you don't need to make any edits to the `dev` settings.